include/zenoh-pico/collections/element.h

```diff
@@ -66,6 +66,11 @@ static inline void _z_noop_copy(void *dst, const void *src) {
     (void)(src);
 }
 
+static inline void _z_noop_move(void *dst, void *src) {
+    (void)(dst);
+    (void)(src);
```

Comment:
Should use the `_ZP_UNUSED` macro for these.

Reply:
fixed